server/users.ts

```diff
@@ -1156,7 +1156,7 @@ export class User extends Chat.MessageContext {
 		this.isStaff = Config.groups[this.group] && (Config.groups[this.group].lock || Config.groups[this.group].root);
 		if (!this.isStaff) {
 			const staffRoom = Rooms('staff');
-			this.isStaff = (staffRoom && staffRoom.auth && staffRoom.auth[this.userid]);
+			this.isStaff = Boolean(staffRoom && staffRoom.auth && staffRoom.auth[this.userid]);
```

Comment:
```suggestion
			this.isStaff = !!(staffRoom && staffRoom.auth && staffRoom.auth[this.userid]);
```

`Boolean` may be more explicit but all the rest of our code uses `!!` – may as well stay consistent until we ever decide to change it all at once.